site/content/en/docs/manual/advanced/filter.md

```diff
@@ -44,7 +44,30 @@ The `Add rule` button adds a rule for objects display. A rule may use the follow
+| -------------- | -------------------------------------------- | ------------------------------------------- |
+| `State`        | all the state names                          | The state of the job <br>(can be changed in the menu inside the job) |
+| `Stage`        | all the stage names                          | The stage of the job <br>(is specified by a drop-down list on the task page) |
+| `Dimension`    | `2D` or `3D`                                 | Depends on the data format <br>(read more in [creating an annotation task](/docs/manual/basics/creating_an_annotation_task)) |
+| `Assignee`     | username                                     | Assignee is the user who is working on the project, task or job. <br>(is specified on task page) |
+| `Owner`        | username                                     | The user who owns the project, task, or job |
+| `Last updated` | last modified date and time (or value range) | The date can be entered in the `dd.MM.yyyy HH:mm` format <br>or by selecting the date in the window that appears <br>when you click on the input field |
+| `Status`       | `annotation`, `validation` or `completed`    |                                             |
+| `Data`         | `video`, `images`                            | Depends on the data format <br>(read more in [creating an annotation task](/docs/manual/basics/creating_an_annotation_task)) |
+| `Subset`       | `test`, `train`, `validation` or custom subset |                                           |
+| `ID`           | number or range of job ID                    |                                             |
+| `Name`         | name                                         | On the tasks page - name of the task,<br> on the project page - name of the project |
+| `Task ID`      | number or range of task ID                   |                                             |
+| `Project ID`   | number or range of project ID                |                                             |
+| `Task name`    | task name                                    | Set when creating a task, <br>can be changed on the ([task page](/docs/manual/basics/task-details/)) |
+| `Project name` | project name                                 | Specified when creating a project, <br>can be changed on the ([project section](/docs/manual/advanced/projects/)) |
+| `Provider type` | `AWS S3`, `Azure`, `Google cloud`           |                                             |
+| `Credentials type` | `Key & secret key`, `Account name and token`,<br> `Anonymous access`, `Key file` |     |
+| `Description`  |                                              | Description of the cloud storage            |
+| `Resource name` |                                             | `Bucket name` or `container name`           |
+| `Display name` |                                              | Set when creating cloud storage             |
```

Comment:
Do not you think it would be better to divide them to different tables? (tasks, projects, jobs, cloud storages). Part of rows will be repeated, but one table looks a bit difficult